package.json

```diff
@@ -21,11 +21,11 @@
   },
   "main": "index.js",
   "dependencies": {
-    "fibers": "^3.1.0",
+    "@wdio/sync": "^5.18.5",
+    "fibers": "^4.0.3",
     "meteor-promise": "^0.8.2",
     "underscore": "^1.8.3",
-    "wdio-sync": "^0.7.0",
     "webdriverio": "^4.14.4",
-    "xolvio-fiber-utils": "3.0.0"
+    "xolvio-fiber-utils": "git+ssh://github.com:TheBrainFamily/fiber-utils#bump-fibers-version"
```

Comment:
This fails with `yarn install` .. need to be "TheBrainFamily/fiber-utils#bump-fibers-version" or "git+ssh://git@github.com:TheBrainFamily/fiber-utils#bump-fibers-version"